src/NServiceBus.Core/Pipeline/PipelineCollection.cs

```diff
@@ -4,12 +4,10 @@ namespace NServiceBus
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
-    using Logging;
+    using NServiceBus.Logging;
```

Comment:
Side note: We need to tweak this setting in R#/VS to not add uneeded NS qualifications.  I'll raise a new issue for this